DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/ViewDelegateControllerTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.TrackingSystem.Delegates
+{
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using FakeItEasy;
+    using FluentAssertions.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Http;
+    using NUnit.Framework;
+
+    class ViewDelegateControllerTests
+    {
+        private ViewDelegateController viewDelegateController = null!;
+        private IUserDataService userDataService = null!;
+
+        [SetUp]
+        public void SetUp()
+        {
+
+            var centreCustomPromptsService = A.Fake<ICentreCustomPromptsService>();
+            var centreCustomPromptsHelper = new CentreCustomPromptHelper(centreCustomPromptsService);
+            userDataService = A.Fake<IUserDataService>();
+            var courseService = A.Fake<ICourseService>();
+            var passwordResetService = A.Fake<IPasswordResetService>();
+
+            var httpRequest = A.Fake<HttpRequest>();
+            var httpResponse = A.Fake<HttpResponse>();
+            const string cookieName = "";
+            const string cookieValue = "";
+
+            viewDelegateController = new ViewDelegateController(userDataService, centreCustomPromptsHelper, courseService, passwordResetService)

```

Comment:
Nice tests! Do we need all this setup though (e.g. the MockTempData, and could HTTPContext potentially be defaultcontext)?